external/lucene-solr/playlist.xml

```diff
@@ -27,7 +27,7 @@
 		</groups>
 	</test>
 	<test>
-		<testCaseName>lucene_solr_nightly_smoketest</testCaseName>
+		<testCaseName>lucene-solr_test</testCaseName>
```

Comment:
'-' dashes need [special handling](https://github.com/adoptium/aqa-tests/blob/master/external/dockerfile_functions.sh#L62-L74) in the shell scripts, so our naming standard should be standardizing to use '_' underscores, so ideally we rename both the directory names and target names to use underscores.